cerberus-web/build.gradle

```diff
@@ -64,9 +64,17 @@ dependencies {
   implementation "com.amazonaws:aws-java-sdk-core:${versions.awsSdkVersion}"
   implementation "com.amazonaws:aws-java-sdk-kms:${versions.awsSdkVersion}"
   implementation "com.amazonaws:aws-java-sdk-sts:${versions.awsSdkVersion}"
+  implementation "com.amazonaws:aws-java-sdk-s3:${versions.awsSdkVersion}"
   implementation "com.amazonaws:aws-java-sdk-secretsmanager:${versions.awsSdkVersion}"
   implementation 'com.amazonaws:aws-encryption-sdk-java:1.6.1'
 
+
+  // JWT
+  implementation "io.jsonwebtoken:jjwt-api:0.10.5"
```

Comment:
These versions are quite old and they fixed a few vulnerabilities since then. Let's at least upgrade to 0.10.8 or newer.